common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -391,11 +391,10 @@ public class CommonValue {
         + "      \""
         + MLTask.IS_ASYNC_TASK_FIELD
         + "\" : {\"type\" : \"boolean\"}, \n"
-        + USER_FIELD_MAPPING
-        + "    }\n"
-        + "}"
+        + "      \""
         + MLTask.REMOTE_JOB_FIELD
-        + "\" : {\"type\": \"flat_object\"}\n"
+        + "\" : {\"type\": \"flat_object\"}, \n"
```

Comment:
if we want to keep the consistency with other fields may be we can add  `+ "      \""` after this?

Reply:
Not quite get you, that will bring extra `\"` which will be a wrong index mapping

Reply:
Sorry the confusion. Please ignore my comment.